src/NuGet.Core/NuGet.PackageManagement/Utility/PackagesConfigLockFileUtility.cs

```diff
@@ -176,11 +267,13 @@ private static void RemoveUninstalledPackages(PackagesLockFile lockFile, IEnumer
             {
                 Debug.Assert(toInstall.NuGetProjectActionType == NuGetProjectActionType.Install);
 
+                const bool includeMinVersion = true;
```

Comment:
why not just declare it class wide?

Reply:
reduces "pollution"

Reply:
ehh, you don't use this anywhere else. 

Using the named parameters is probably better :) 

I'm not gonna block on this though. :)

Reply:
As you guessed, I added it only because named parameters when you don't name every parameter after the first is a newer c# feature that our CI pipeline didn't support when I started this work. Since our pipeline uses VS2019 now and I think we're using this feature elsewhere, it feels safe enough to use. Pushing an update soon.